src/wirecloud/platform/iwidget/models.py

```diff
@@ -0,0 +1,89 @@
+# -*- coding: utf-8 -*-
+
+# Copyright (c) 2012-2016 CoNWeT Lab., Universidad Politécnica de Madrid
+
+# This file is part of Wirecloud.
+
+# Wirecloud is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+
+# Wirecloud is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+
+# You should have received a copy of the GNU Affero General Public License
+# along with Wirecloud.  If not, see <http://www.gnu.org/licenses/>.
+
+
+from pydantic import BaseModel, Field, StringConstraints
+from typing import Annotated, Any, Optional
+from enum import Enum
+
+
+Id = str
+IntegerStr = Annotated[str, StringConstraints(pattern=r'^\d+$')]
+
+
+class DBWidgetConfigAnchor(BaseModel, use_enum_values=True):
+    top_left = "top-left"
+    top_center = "top-center"
+    top_right = "top-right"
+    bottom_left = "bottom-left"
+    bottom_center = "bottom-center"
+    bottom_right = "bottom-right"
+
+
+class DBWidgetConfig(BaseModel):
```

Comment:
El `use_enum_values` es donde se use el enum, no en la definición.
```suggestion
class DBWidgetConfig(BaseModel, use_enum_values=True):
```